vividus-plugin-web-app-to-rest-api/src/test/java/org/vividus/steps/integration/ResourceCheckStepsTests.java

```diff
@@ -485,10 +499,12 @@ void shouldFilterResourceByRegExpCheckDesiredResourcesAnPostAttachment()
 
     private void runExecutor() throws InterruptedException, ExecutionException
     {
-        doNothing().when(executor).execute(argThat(r -> {
```

Comment:
junit fails with unnecessary stubbing if we throw an exception inside executor, so changed to doAnswer